modules/lang-painless/src/main/java/org/elasticsearch/painless/Definition.java

```diff
@@ -518,15 +517,15 @@ public RuntimeClass getRuntimeClass(Class<?> clazz) {
     private final Map<String, Struct> structsMap;
     private final Map<String, Type> simpleTypesMap;
 
-    private Definition() {
+    private Definition(List<String> definitionFiles) {
```

Comment:
I'm not sure this is how this'll end up, but it is closer to what we want in the end I think.